### DISCUSSION TITLE
When email settings are applied, no password reset link appears

### DISCUSSION THREAD

**Discussion by ajb3932** [danny-avila/LibreChat#3993] [Troubleshooting] [answered]
Hey,
Amazing app, thank you for your dev time!
I am trying to enable password reset via email on Unraid. I have set the parameters in the Docker settings:

![image](https://github.com/user-attachments/assets/3cbad538-53ff-4879-b7ec-a1145fdb73ec)

But when I load the app there is no reset password link:

![image](https://github.com/user-attachments/assets/db2a9883-5b18-41c1-ad96-0ad03308101f)

Am I missing something in my config? (I tried with Email registration set to true and false)

Thanks

Alex

**Comment by danny-avila** [comment]
> Amazing app, thank you for your dev time!

You're welcome!

I'm not too familiar with Unraid, and I've moved this to discussions since it's not an officially recommended deployment.

Do review the docs on the variables that determine showing the "Reset Password" option.
https://www.librechat.ai/docs/configuration/authentication

If you have any further questions let me know.

**Reply by danny-avila** [reply, marked as answer]
I realized that exact page doesn't show `ALLOW_PASSWORD_RESET` which is required for this. 

https://www.librechat.ai/docs/configuration/dotenv#registration-and-login

**Comment by ajb3932** [comment]
![image](https://github.com/user-attachments/assets/e2641101-adeb-4bcb-9fb4-9bda86062617)
Perfect, thank you so much!
